### ISSUE TITLE
I can only get 43.61% accuracy in Somethingv1

### ISSUE THREAD

**Issue by dreamer121121** [chenxuluo/GST-video#4]
I can only get 43.61% Accuracy in somethingv1, limited by my hardware i only use one GPU with batch-size = 16. why i cant get ur result ? Here is my train loss curve 
![trainLoss](https://user-images.githubusercontent.com/35915584/81521704-9434c180-937a-11ea-9228-3183c89b7af8.png)
Im looking forward to ur reply

**Comment by chenxuluo** [comment]
I'm not sure what happens. Maybe the batch-size matters. It shouldn't get so bad results using the default setting. Also you need to pay attention to the augmentation (https://github.com/chenxuluo/GST-video/blob/master/main.py#L38) if you use different label orders.

**Comment by dreamer121121** [comment]
thank u for ur reply.i use the process_dataset.py to get the category.txt,train.txt and the val.txt. so i think the label orders is not the reason. All configs of training is identical to ur papaer.

**Comment by dreamer121121** [comment]
Im sorry to desturb u again，I train ur model with  the bathch = 24 again , the accuracy can got 45.6% in somethingv1, a little higher than last time ,but I still cannot got ur result .Can u open source  ur trained model?  This is my wechat:x18215589257. Thank u

**Comment by chenxuluo** [comment]
You can find the pretrained model here:  https://drive.google.com/file/d/1q8pI3w-GnCT34hfTTCIm0TQJJaq9sT5B/view?usp=sharing
To make it clear, the current transformation in the code is based on the label order provided in sth-v2.  However, the pretrained model is based on the label orders in v1.

**Comment by slighting666** [comment]
@dreamer121121 
我利用GST.PY这个文件复现了resnet的代码，并利用ucf-101数据集进行测试，但是总是出现这样的错误。
The size of tensor a (28) must match the size of tensor b (14) at non-singleton dimension 4
你能帮帮我吗？十分感谢。如果您不嫌麻烦，能否加我qq：858856173
谢谢
